### ISSUE TITLE
Drupal.org Project?

### ISSUE THREAD

**Issue by q0rban** [e0ipso/shapeshifter#5]
Any plans to release this on d.o?

**Comment by e0ipso** [comment]
Shapeshifter has been superseded by RESTful. I forked the data extraction part out of RESTful to create Shapeshifter, but it was not a very good move since I could not keep up with the maintenance of both.

RESTful is Shapeshifter + a dynamic hook_menu. That hook menu can be disabled via config, making RESTful into Shapeshifter. That's why I decided to abandon the module development and encourage using RESTful instead. There's no practical overhead to use RESTful instead of Shapeshifter.

Long story short, this repo is a good example on an unexpected, but practical, use of RESTful but I don't have plans to release it to d.o.

**Comment by q0rban** [comment]
I see!
